scripts/docker/install_mysql.sh

```diff
@@ -50,6 +50,7 @@ if [[ ${INSTALL_MYSQL_CLIENT:="true"} == "true" ]]; then
     gpgconf --kill all
     rm -rf "${GNUPGHOME}"
     apt-key list > /dev/null
+    echo "deb http://repo.mysql.com/apt/debian/ buster mysql-5.7" | tee -a /etc/apt/sources.list.d/mysql.list
```

Comment:
I think MYSQL8 client libs should work with both 5.7 and 8 - MySQL client libraries are meant to be fully backwards compatible - in v1-10-test we have now 5.7 client libraries only and they work fine with both 5.6 and 5.7 versions.

Reply:
But yeah.. Maybe that will help :)

Reply:
Yeah I tried locally, basically, the `libmysqlclient20` was MySQL 5.7 only so the "Build image" step was failing initially. But should work now 🤞 .

I am just trying to keep both but yeah, might remove 5.7 if it is totally compatible

Reply:
I think only one will be used anyway if both are installed with both DBs (not sure which one though)